packages/dev-react/config/tslint.js

```diff
@@ -3,5 +3,9 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 module.exports = {
-  'extends': '@polkadot/dev/config/tslint'
+  extends: '@polkadot/dev/config/tslint',
+  rulesDirectory: './node_modules/tslint-react/rules/',
```

Comment:
Can it extend an array, and remove this line?

Reply:
Then it adds all of them from `tslint-react`. So rather that overriding/disabling all, just trying to pull in the single rule

Reply:
... and couldn't find another way to pull in a single rule, `extends` pulls in all

Reply:
Ah yeah true